docs/reuse/apps/app-install-v2.md

```diff
@@ -8,9 +8,7 @@ To install the app, do the following:
     :::
 1. Click **Next** in the **Setup Data** section.
 1. In the **Configure** section of your respective app, complete the following fields.
```

Comment:
Create seperate files for different types of installation

1. sc k8s - apache
2. without sc, k8s - mysql
3. sc, without k8s - ingress